spark-common/src/main/java/me/lucko/spark/common/sampler/java/TickedDataAggregator.java

```diff
@@ -151,6 +151,17 @@ public List<ThreadInfo> getList() {
             return this.list;
         }
 
+        public int sizeWithoutTrailingSleeping() {
```

Comment:
Is there a reason this method uses `this.list` instead of `currentData.getList()` (could pass it as a parameter?)

Reply:
I just put the method into `TickList`, but there isn't much thought to it. I can also move it to the enclosing class if you want.

Reply:
Ahhh doh, sorry I misread - it's good how it is :)